### PR TITLE
Codegen for openapi 6f9da2d

### PR DESCRIPTION
**Pull request by richardm-stripe**
Codegen for openapi 6f9da2d.
r? @remi-stripe 
cc @stripe/api-libraries

## Changelog
* Add support for Identity VerificationSession and VerificationReport APIs

### CODE REVIEW

File: src/Stripe.net/Entities/Identity/VerificationReports/VerificationReport.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        /// String representing the object's type. Objects of the same type share the same value.
+        /// </summary>
+        [JsonProperty("object")]
+        public string Object { get; set; }
+
+        /// <summary>
+        /// Time at which the object was created. Measured in seconds since the Unix epoch.
+        /// </summary>
+        [JsonProperty("created")]
+        public long Created { get; set; }
```
Comment (remi-stripe) [inline src/Stripe.net/Entities/Identity/VerificationReports/VerificationReport.cs:24]: This is the wrong type right? Should it be a DateTime like https://github.com/stripe/stripe-dotnet/blob/master/src/Stripe.net/Entities/Charges/Charge.cs#L185

File: src/StripeTests/Entities/Identity/VerificationReports/VerificationReportTest.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            Assert.NotNull(verificationReport.Id);
+            Assert.Equal("identity.verification_report", verificationReport.Object);
+        }
+
+        [Fact]
+        public void DeserializeWithExpansions()
+        {
+            string[] expansions =
+            {
+              "cardholder",
```
Comment (remi-stripe) [inline src/StripeTests/Entities/Identity/VerificationReports/VerificationReportTest.cs:30]: That test is incorrect, there's no cardholder on VR, why doesn't this error?

File: src/StripeTests/Entities/Identity/VerificationSessions/VerificationSessionTest.cs [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            Assert.NotNull(verificationSession.Id);
+            Assert.Equal("identity.verification_session", verificationSession.Object);
+        }
+
+        [Fact]
+        public void DeserializeWithExpansions()
+        {
+            string[] expansions =
+            {
+              "cardholder",
```
Comment (remi-stripe) [inline src/StripeTests/Entities/Identity/VerificationSessions/VerificationSessionTest.cs:30]: Same here

**Review by remi-stripe — CHANGES_REQUESTED** [review]

File: src/StripeTests/Entities/Identity/VerificationReports/VerificationReportTest.cs (hunk 2 above)
Comment (richardm-stripe) [reply src/StripeTests/Entities/Identity/VerificationReports/VerificationReportTest.cs:30]: Great catch. VR has no expansions so I've removed this test. The "not-failing" behavior originates in stripe-mock. I really can't make sense of stripe-mock's behavior here:

`curl localhost:12111/v1/identity/verification_sessions?expand[]=customer -u sk_test_xzy:` doesn't throw a validation error, but `curl localhost:12111/v1/identity/verification_sessions?expand[]=foo -u sk_test_xzy:` throws a 500.

In any case, the test is fixed but the underlying issue is not.

**Review by remi-stripe — APPROVED** [review]

File: src/Stripe.net/Constants/Events.cs [hunk 4]
```diff
@@ -275,6 +275,28 @@ public static class Events
...
+        /// Occurs whenever a VerificationSession is created.
+        /// </summary>
+        public const string IdentityVerificationSessionCreated = "identity.verification_session.created";
+
+        /// <summary>
+        /// Occurs whenever a VerificationSession transitions to processing.
+        /// </summary>
+        public const string IdentityVerificationSessionProcessing = "identity.verification_session.processing";
+
+        public const string IdentityVerificationSessionRedacted = "identity.verification_session.redacted";
```
Comment (remi-stripe) [inline src/Stripe.net/Constants/Events.cs:293]: missing docs

File: src/Stripe.net/Constants/Events.cs [hunk 5]
```diff
@@ -275,6 +275,28 @@ public static class Events
...
+
+        /// <summary>
+        /// Occurs whenever a VerificationSession transitions to require user input
+        /// </summary>
+        public const string IdentityVerificationSessionRequiresInput = "identity.verification_session.requires_input";
+
```
Comment (remi-stripe) [inline src/Stripe.net/Constants/Events.cs:299]: Missing the verified event

**Review by remi-stripe — CHANGES_REQUESTED** [review]

File: src/StripeTests/Entities/Identity/VerificationReports/VerificationReportTest.cs [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    {
+        public VerificationReportTest(StripeMockFixture stripeMockFixture)
+            : base(stripeMockFixture)
+        {
+        }
+
+        [Fact]
+        public void Deserialize()
+        {
+            string json = this.GetFixture("/v1/identity/verification_reports/vs_123");
```
Comment (remi-stripe) [inline src/StripeTests/Entities/Identity/VerificationReports/VerificationReportTest.cs:17]: minor but this should be vr_123

File: src/Stripe.net/Constants/Events.cs (hunk 5 above)
Comment (remi-stripe) [reply src/Stripe.net/Constants/Events.cs:307]: @richardm-stripe that one should be above `verified`

**Review by remi-stripe — APPROVED** [review]

**Pull request merged**